### PR TITLE
Add contacts shimmer and improve link-based conversation creation items

### PR DESCRIPTION
**Pull request by AndyScherzinger**
* We now show a simmer effect like we do for conversations
* start conversations via link(s) now uses a circular icon

:sunny:start conversation (before)|:new_moon:start conversation (before)|:sun_with_face:start conversation (after)|:new_moon_with_face:start conversation (after)|:new_moon_with_face:shimmer
---|---|---|---|---
before_light|before_dark|light|dark|shimmer_dark

Signed-off-by: Andy Scherzinger <info@andy-scherzinger.de>

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
@mahibi please test before merging :9 When I did the same change some weeks ago I had a lot of whitespace at the top of the list. While I can't say due to all the lib updates if this has been fixed somewhere automagically...

**Review by jancborchardt — APPROVED** [review]
Nice, much better! :)

Possibly for a follow-up:
- The "New conversation" would be better as the first entry, since "Join via link" is by far not as common
- And the icon for "New conversation should be something else too. Possibly the contacts/group icon

**Comment by AndyScherzinger** [comment]
@jancborchardt if anyone can tell me where to find the group icon svg, than I can add this here :) Not a bit change

**Comment by AndyScherzinger** [comment]
Nevermind, found the icon.

@jancborchardt did the changes right here:
review

**Comment by nickvergessen** [comment]
Person icon and share icon should be on grey background like the final result when you create such rooms?

File: app/src/main/java/com/nextcloud/talk/controllers/ContactsController.java [hunk 1]
```diff
@@ -658,12 +665,22 @@ private void prepareViews() {
             }
         });
 
+        joinConversationViaLinkImageView
+                .getBackground()
+                .setColorFilter(ResourcesCompat.getColor(getResources(), R.color.colorPrimary, null),
```
Comment (nickvergessen) [inline app/src/main/java/com/nextcloud/talk/controllers/ContactsController.java:670]: Should use the grey (server name: `color-background-darker`) which is `#dbdbdb` in bright and `#2c2c2c`  in dark mode

Comment (AndyScherzinger) [reply app/src/main/java/com/nextcloud/talk/controllers/ContactsController.java]: DONE

Comment (AndyScherzinger) [reply app/src/main/java/com/nextcloud/talk/controllers/ContactsController.java]: https://github.com/nextcloud/talk-android/pull/1209/commits/79e5c3bcaff7e18050da381a125a54a17a9bcb92

**Comment by AndyScherzinger** [comment]
dark|light
---|---
dark|light

**Review by mahibi — APPROVED** [review]

**Comment by mahibi** [comment]
> @mahibi please test before merging :9 When I did the same change some weeks ago I had a lot of whitespace at the top of the list. While I can't say due to all the lib updates if this has been fixed somewhere automagically...

haven't seen any whitespaces, seems all good to me..

**Pull request merged**